lua/config/defaults.lua

```diff
@@ -20,7 +20,7 @@ lvim = {
     level = "warn",
     viewer = {
       ---@usage this will fallback on "less +F" if not found
-      cmd = "lnav",
+      cmd = "less +F",
```

Comment:
Sorry I didn't make it clear. It's better to set it like this:

```lua
local log_viewer = lvim.log.viewer.cmd
if vim.fn.executable(log_viewer) ~= 0 then
	log_viewer = 'less +F'
end
local cmd = log_viewer .. " " .. logfile
local term_opts = vim.tbl_deep_extend("force", lvim.builtin.terminal, {
    cmd = cmd,
	--- rest of the arguments
```

in here:
https://github.com/LunarVim/LunarVim/blob/6b79e00cb04bf889fc30b84482d6db6e951ca9e9/lua/core/terminal.lua#L115-L116

Reply:
Sorry for my silly mistake. Solved now.